src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -979,7 +981,7 @@ protected function getDefaultValueDefinition($default)
      */
     protected function getColumnSqlDefinition(Column $column)
     {
-        $sqlType = $this->getSqlType($column->getType());
+        $sqlType = $this->getSqlType($column->getType(), $column->isIdentity());
```

Comment:
This would be the better place to make the modification... If `$column->getType()` is `PHINX_TYPE_BIG_INTEGER` and `$column->isIdentity()`, then send `PHINX_TYPE_INTEGER` to `getSqlType` instead.